api/tests/python/end_points/test_requests.py

```diff
@@ -292,4 +292,93 @@ def test_remove_name_from_nr(client, jwt, app):
 
     data = json.loads(rv.data)
     assert 200 == rv.status_code
-    assert len(data['names']) == 1
\ No newline at end of file
+    assert len(data['names']) == 1
+
+    def test_add_new_comment_to_nr(client, jwt, app):
+        from namex.models import Request as RequestDAO, State, Name as NameDAO, Comment as CommentDAO, User
```

Comment:
Looks like this line and next are indented incorrectly (so they're part of previous test, ie: not running) - like 297 ad 298 need to be pulled back by one tab each.

Reply:
Fixed.  Thanks.  Sorry.

Reply:
ok. fixed both of them this time.